src/react-magic-script/magic-script-renderer.js

```diff
@@ -413,10 +414,7 @@ const HostConfig = {
 };
 
 function logNotImplemented(functionName) {
-  const message = `${functionName} has not been implemented yet`;
-
-  console.log(message);
-  print(message);
+  log(`${functionName} has not been implemented yet`, MessageSeverity.warning);
```

Comment:
`MessageSeverity.warning`, etc. is pretty verbose. Since we will probably be using this a lot, maybe we can adopt something similar to Android as a shorthand:

-  log.i('info message')
-  log.w('warning message')
-  log.e('error message')
-  log.x('exception message')

These could all just invoke logWithSeverity or some similarly named function.

The exact naming is just a suggestion...I'm really just hoping for something more concise.